### ISSUE TITLE
small suggestion.

### ISSUE THREAD

**Issue by The-Nyla** [TheDutchMC/rConsole#2]
Would you be able to add color code support for rconsle? also it would be nice if you could add some buttons on the side that you could assign specific commands to, like /weather clear (without the "/" of course) if you used that command often, essentially just one-click use of frequently used commands that the user defines.

**Comment by TobiasDeBruijn** [comment]
Color codes should work for ASCII colors, e.g /help produces colored output. I'm thinking you mean colors as in &<code>?

Command 'presets' is a thing I could definitely include yep, shouldn't be too much work. I cannot give an ETA for it yet though, as I first want to get to a 1.0.0 release

**Comment by The-Nyla** [comment]
Yes, I did mean &? and thank you! I can't wait for the 1.0.0 release, this plugin, with it's current features, is already one of the best plugins I've ever used.

**Comment by The-Nyla** [comment]
wait, should I have closed this?

**Comment by TobiasDeBruijn** [comment]
Nope :"D Please keep it open until the feature is implemented :D

**Comment by TobiasDeBruijn** [comment]
> Yes, I did mean &? and thank you! I can't wait for the 1.0.0 release, this plugin, with it's current features, is already one of the best plugins I've ever used.

Thanks :"D Really means a lot !
